src/views/start/start.js

```diff
@@ -1,13 +1,117 @@
+import {
+	useRef,
+	useState,
+} from "react";
+
+import classNames from "classnames";
+import { Link } from "react-router-dom";
+
+import { Avatar } from "../../components/Avatar/Avatar";
+import { Button } from "../../components/Button/Button";
+import {
+	useGameContext,
+	useGameDispatch,
+} from "../../components/GameProvider/GameProvider";
+import { Input } from "../../components/Input/Input";
 import { Logo } from "../../components/Logo/Logo";
-import styles from "./_start.module.css";
+import { games } from "../../utils/constants";
+import style from "./_start.module.css";
 
 export function Start() {
+	const inputRef = useRef();
+	const dispatch = useGameDispatch();
+	const gameState = useGameContext();
+
+	const [inputVal, setInputVal] = useState('');
+
+	const isLinkDisabled = Object.values(gameState.players).length >= 2 && !!gameState.selectedGame;
+	
+	const linkClassName = classNames(style.link,
+		{[style.linkDisabled] : !isLinkDisabled });
+
+	const handlePlayerAdd = (event) => {
+		event.preventDefault();
+		dispatch({
+			type: 'ADD_PLAYER',
+			payload: inputRef.current.value
+		});
+		setInputVal(!inputVal);
+		inputRef.current.value = '';
+	}
+
+	const handlePlayerRemove = (data) => {
+		dispatch({
+			type: 'REMOVE_PLAYER',
+			payload: data
+		})
+	}
```

Comment:
Dodeljujete opisnija imena za argumente fukcija.
```suggestion
	const handlePlayerRemove = (playerId) => {
		dispatch({
			type: 'REMOVE_PLAYER',
			payload: playerId
		})
	}
```